setup.py

```diff
@@ -32,7 +32,6 @@
     'Flask-Bcrypt==0.6.2',
     'Flask-Principal==0.4.0',
     'Flask-Mail==0.9.1',
-    'SQLAlchemy-Utils==0.30.11',
```

Comment:
It looks like I also use the JSON type from SQLAlchemy-Utils so we should leave it in for the moment.